build.gradle

```diff
@@ -423,6 +423,17 @@ task integrationTest(type: Test) {
 	environment(run.environment)
 }
 
+// Create the task that runs the migration script for empty HTML biography fields of persons
+task dbClearBiographies(dependsOn: 'compileJava', type: JavaExec) {
```

Comment:
I suggest to rename to dbClearEmptyBiographies as current name is misleading

Reply:
I agree, changed that.